packages/push-notifications/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import 'reflect-metadata'
 
 export { PushNotificationsApnsModule, PushNotificationsFcmModule } from './modules'
+export { PushNotificationsApnsApi, PushNotificationsFcmApi } from './api'
```

Comment:
Is there a reason why you would import the Api and not the Module?

Reply:
sorry i'm not entirely sure what you mean here

Reply:
ah thx for explaining. I see what you mean now. yes, let's remove the export